docs/data/catalog/platform/gc-forms/templates.md

```diff
@@ -31,7 +31,7 @@ This dataset is extracted daily from the GC Forms database's `Templates` and `Us
 
 * `Updated`: Daily
 * `Steward`: GC Forms and Platform Core Services
```

Comment:
same for Steward, I'll just list GC Forms

Reply:
whoops my bad - thanks for catching and fixed now